cmd/registry/patch/apply.go

```diff
@@ -66,7 +66,8 @@ func (p *patchGroup) add(task *applyFileTask) error {
 	}
 	header, err := readHeader(bytes)
 	if err != nil {
-		return err
+		// Skip YAML files that don't have our expected headers. We assume they aren't ours.
+		return nil
```

Comment:
That would be tricky - if we log every YAML file that isn't one of ours, we would log all the openapi.yaml files in a hierarchy. I'm not sure how to do that without being tedious or confusing.

If you're concerned that we would fail silently on some erroneous intended inputs, I agree, and can put this in an issue.

Reply:
This depends on use case expectations... I didn't expect users would be mixing our YAML files with arbitrary ones. Seems risky - especially if we're not logging what gets skipped.

The other issues is, as you noted, an incorrect header format that was intended for us. Beside bad formatting, one of the error conditions is simply an [incorrect version](https://github.com/apigee/registry/blob/main/cmd/registry/patch/header.go#L33). How would the user know how to correct these?

Thanks, opening an issue seems wise.